frontend/app/routes/public/renew/$id/adult/confirmation.tsx

```diff
@@ -154,7 +154,7 @@ export default function RenewAdultConfirm() {
             event.preventDefault();
             window.print();
           }}
-          data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Adult:Print top - Application successfully submitted click"
+          data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Adult:Print top - Your renewal for the Canadian Dental Care Plan is complete click"
```

Comment:
(Same comment applies for the rest of the attribute values in this route)

```suggestion
          data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult:Print top - Your renewal for the Canadian Dental Care Plan is complete click"
```